salt/channel/server.py

```diff
@@ -113,6 +140,84 @@ def post_fork(self, _, __):
 
         self.master_key = salt.crypt.MasterKeys(self.opts)
 
+    @salt.ext.tornado.gen.coroutine
+    def handle_message(self, stream, payload, header=None):
+        stream = self.transport.wrap_stream(stream)
```

Comment:
what type of object is "stream" at this point? A tornado stream? Would this concept be applicable to broker-based transports

Reply:
We've removed stream from this method. It passes in the message and the method should return the reply.